app/views/handlers/list_repeating_block.py

```diff
@@ -0,0 +1,49 @@
+from functools import cached_property
+
+from flask import url_for
+
+from app.views.handlers.list_edit_question import ListEditQuestion
+
+
+class ListRepeatingBlock(ListEditQuestion):
+    @cached_property
+    def repeating_block_ids(self) -> list[str]:
+        return [block["id"] for block in self.parent_block["repeating_blocks"]]
```

Comment:
Any reason why you're not reusing the map you populated upfront in questionnaire schema here -`repeating_blocks_by_id`?

Reply:
Great shout, will do that now